proxy_mobile/react-native/ReactNativeApp.js

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import { StyleSheet, View } from 'react-native';
 import { Button } from 'react-native';
 import { ARView } from 'magic-script-components-react-native';
+import { setInitialDeeplink } from '../../global/globalVariables'
 
 export default class ReactNativeApp extends React.Component {
-  
+
   constructor(props) {
     super(props);
-    this.state = { planeDetection: true} ;
+    this.state = { planeDetection: true };
+    
+    if(props.initialUrl != null) {
```

Comment:
Do we need to check `initialUrl` agains null/undefined?
BTW In .js we should use `!==/===` for comparison ([SO link](https://stackoverflow.com/questions/2778901/how-to-compare-variables-to-undefined-if-i-don-t-know-whether-they-exist)).

Reply:
in my opinion, there is no need to notify observers if there is no `initialUrl`